src/dreyfus_httpd.erl

```diff
@@ -443,6 +444,13 @@ validate_search_restrictions(Db, Args) ->
             ok
     end,
 
+    case fabric_util:is_partitioned(Db) of
```

Comment:
Maybe we should put these into the config so that we can make them customizable rather than hardcoded like this.

Reply:
Originally, `max_limit` is hardcoded, and followed the same way for `max_limit_partitions`. However, good idea to make it configurable and changed with `8caff576768aeabdf38b7e48aaaf67078e849ff3`